### ISSUE TITLE
sass importing problem

### ISSUE THREAD

**Issue by basz** [kaliber5/ember-bootstrap#1871]
Not sure if this is an eb issue, but seems related.

I have a project that I am adding ember-bootstrap to. Since i want to be picky about what i use of bootstrap i do some manual imports of scss files. However even while I can `@import "ember-bootstrap/variables";` I am unable to `@import "ember-bootstrap/functions";` which is required due to a "File to import not found or unreadable" error. I checked the file is there and readable. Also trashed node_modules several times.

I do not understand why? When I copy all bootstrap/scss/* file to my styles I *can* import each and every one of them.

So since I am using pnpm and a monorepo I am suspecting these as an possible issue. Yet I have a similar setup in other projects. I also have played with hoisting configuration via .npmrc... I have added the bootstrap package to the relevant workspace package *and* to the root package. adding bootstrap to the root package and using the following import works; `@import "./node_modules/bootstrap/scss/functions";`

None of these seem to work... To complement the complexity this is a project setup for tailwind, so that seems to use a something different to load scss files.

To recap;

copying all bootstrap/scss files into my project and then importing them compiles works.
adding bootstrap to the root package and using this import works; `@import "./node_modules/bootstrap/scss/functions";`
I cannot import adding bootstrap to the root package and using this import also works; `@import "bootstrap/functions";` but `@import "bootstrap/variables";` does work..

😕

**Comment by jelhan** [comment]
> To complement the complexity this is a project setup for tailwind, so that seems to use a something different to load scss files.

Ember Bootstrap only supports `ember-cli-sass` as preprocessor: https://github.com/ember-bootstrap/ember-bootstrap/blob/f4b6cefe1514ac59262bb94f783af179cbc8b3a7/index.js#L19 https://github.com/ember-bootstrap/ember-bootstrap/blob/f4b6cefe1514ac59262bb94f783af179cbc8b3a7/index.js#L130-L132

If you are using a different tool to integrate SASS into your Ember build, you likely need to do the integration manually.
